src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] it would be better to  remove the disable linter lines from all the files in order to keep the code clean and pro

Reply:
I removed the linter blocker from the file as per requested